metabase/metabase
# Migrate SDK e2e tests to Cypress Component tests

Part of https://github.com/metabase/metabase/issues/50113

### Description

This migrates most of SDK tests to be Cypress Component tests
Runs migrated tests SDK built from the latest source code.

### How to verify

Check CI results
Run tests locally from this branch - `yarn test-cypress-open-component-sdk` (you will also need EE env vars for Cypress)

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist
